### PR TITLE
Travis CI build setup

### PR DESCRIPTION
**Pull request by oalyman**
Travis CI config and gradle build adjustment to be able to build / test on Travis CI.

### CODE REVIEW

**Comment by clemensutschig** [comment]
Super cool!

**Comment by clemensutschig** [comment]
@oalyman  - can you merge yourself?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+language: java
+install: true
+
+jdk:
+  - oraclejdk8
```
Comment (tjaeschke) [inline .travis.yml]: Just a question. Shouldn't we use openjdk instead of oraclejdk?

Comment (oalyman) [reply .travis.yml]: you're right. Thanks for the hint!

**Pull request merged**